components/widgets/list/WidgetCard.js

```diff
@@ -164,6 +170,10 @@ class WidgetCard extends React.Component {
     }
   }
 
+  onVisibilityChange = visibility => () => {
```

Comment:
Are you using it?

Reply:
Of course, it's the button's action.

Reply:
Let me insist, you have this function for the button's action. Ok... but the action doesn't do anything but changing the state... Are you using `collectionPanelVisibility` since you remove `visibility` from Tooltip in other places?

Reply:
ok, ok, I just forgot the tooltip manage its state itself. Removed.